Source/Analytics/Read/Map/CaseReportsBeforeDayQuery.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Linq;
+using Dolittle.Queries;
+using Dolittle.ReadModels;
+
+namespace Read.Map
+{
+    public class CaseReportsBeforeDayQuery : IQueryFor<CaseReportsBeforeDay>
+    {
+        readonly IReadModelRepositoryFor<CaseReportsBeforeDay> _repositoryForCaseReportsBeforeDay;
+
+        public CaseReportsBeforeDayQuery(IReadModelRepositoryFor<CaseReportsBeforeDay> repositoryForCaseReportsBeforeDay)
+        {
+            _repositoryForCaseReportsBeforeDay = repositoryForCaseReportsBeforeDay;
+        }
+
+        public IQueryable<CaseReportsBeforeDay> Query => _repositoryForCaseReportsBeforeDay.Query;
```

Comment:
This should get for today, not all entries. You can add a `.Where(report => report.Id == Day.Today);` to your Query to only get today's data.